pageserve/templates/events.html

```diff
@@ -1,26 +1,39 @@
-<html lang="en">
-    <head>
+{% extends "base.html" %}
+<!--
+Copyright 2019 The Knative Authors
 
-    </head>
-    <body>
-        <h1>Sub-Events</h1>
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
 
-        {% if auth %}
-        <p>user has edit authorization</p>
-        {% else %}
-        <p>user does not have edit authorization</p>
-        {% endif %}
+    http://www.apache.org/licenses/LICENSE-2.0
 
-        <hr>
-        {% for event in events %}
-        <p>Event name: {{event.name}}</p>
-        <p>Event time: {{event.event_time}}</p>
-        <p>Event description: {{event.description}}</p>
-        <p>Created by {{event.author}} at {{event.created_at}}</p>
-        <hr/>
-        {% endfor %}
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
 
-        <a href='/v1/'>See all posts</a>
+{% block header %}
+<h1>{% block title %}Sub-Events{% endblock title %}</h1>
+{% endblock header %}
 
-    </body>
-</html>
+{% block content %}
+{% if auth %}
+<p>user has edit authorization</p>
+{% else %}
+<p>user does not have edit authorization</p>
+{% endif %}
+
+<hr>
+{% for event in events %}
+<p>Event name: {{event.name}}</p>
+<p>Event time: {{event.event_time}}</p>
+<p>Event description: {{event.description}}</p>
+<p>Created by {{event.author}} at {{event.created_at}}</p>
+<hr />
```

Comment:
Why is this `<hr />` and not `<hr>`?

Reply:
I think I originally added this, since they do the same thing we can just change to all `<hr>` for consistency

Reply:
I just copied that from what was there before, but `<hr>` should be sufficient. Done https://github.com/knative-portability/large-events/pull/28/commits/5bee11729e50e2b316084b3be2ed8024c539a788